docs/src/tutorials/algorithms/benders_decomposition.jl

```diff
@@ -266,7 +266,7 @@ function my_callback(cb_data)
         return
     end
     cut = @build_constraint(θ >= ret.obj + -ret.π' * A_1 * (x .- x_k))
-    MOI.submit(model, MOI.LazyConstraint(cb_data), cut)
+    MOI.submit(lazy_model, MOI.LazyConstraint(cb_data), cut)
```

Comment:
I should look at why this didn't error. It should have...

Reply:
![image](https://github.com/jump-dev/JuMP.jl/assets/143272683/07f78241-a0e4-453f-91e9-7006e1ac1115)
It has indeed, after changing GLPK to Gurobi. (I have not installed GLPK)

Reply:
Ah, so this might be an issue with GLPK, not with JuMP. Thanks for looking.